src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLogger.cs

```diff
@@ -31,186 +26,41 @@ internal ConsoleLogger(string name, ConsoleLoggerProcessor loggerProcessor)
             _queueProcessor = loggerProcessor;
         }
 
+        internal ConsoleFormatter Formatter { get; set; }
         internal IExternalScopeProvider ScopeProvider { get; set; }
 
         internal ConsoleLoggerOptions Options { get; set; }
 
+        [ThreadStatic]
+        internal static StringWriter _stringWriter;
```

Comment:
Why is this `internal`? Does it have to be?

Also, since it is thread static, it should be named with `t_` prefix. See (3) in https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md